nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -92,13 +92,14 @@ fi
 mkdir -m 0755 -p $mountPoint/dev $mountPoint/proc $mountPoint/sys $mountPoint/etc $mountPoint/run $mountPoint/home
 mkdir -m 01777 -p $mountPoint/tmp
 mkdir -m 0755 -p $mountPoint/tmp/root
-mkdir -m 0755 -p $mountPoint/var
+mkdir -m 0755 -p $mountPoint/var/permissions-wrappers
```

Comment:
I'm fairly certain it was in this nixos-install.sh script in the 16.03 release but now that I've merged against the 17.03 series it is probably unnecessary, I need to investigate.